src/components/v1/FundingCycle/FundingCycleDetails.tsx

```diff
@@ -1,7 +1,6 @@
-import { ExclamationCircleOutlined } from '@ant-design/icons'
 import { parseEther } from '@ethersproject/units'
```

Comment:
No functional changes here, mostly:
- replacing `t` with `Trans`
- moving complexity out of the main template.
-